tests/test_routes.py

```diff
@@ -134,10 +131,29 @@ def test_get_item(self):
         self.assertEqual(data["restock_level"], test_item.restock_level)
         self.assertEqual(data["condition"], test_item.condition)
 
-    def test_get_item_not_found(self):
-        """It should not Get a Item thats not found"""
-        response = self.client.get(f"{BASE_URL}/0")
+    # def test_get_item_not_found(self):
+    #     """It should not Get a Item thats not found"""
+    #     response = self.client.get(f"{BASE_URL}/0")
+    #     self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+    #     data = response.get_json()
+    #     logging.debug("Response data = %s", data)
+    #     self.assertIn("was not found", data["message"])
+
+    # ----------------------------------------------------------
+    # TEST Delete
+    # ----------------------------------------------------------
+    def test_delete_inventory(self):
```

Comment:
I think 'item' would be more appropriate than 'Inventory' in the method name and functionality